src/CMakeLists.txt

```diff
@@ -323,12 +325,12 @@ else()
 		endif(HAVE_LIBRT)
 	endif(APPLE)
 
-	if(NOT HAIKU AND NOT APPLE)
+	if(NOT HAIKU AND NOT APPLE AND NOT (CMAKE_SYSTEM_NAME MATCHES "kFreeBSD.*|FreeBSD"))
```

Comment:
Can't we just use `if (DEFINED ENV{DISPLAY})` to detect whether X11 is in use or not?

Reply:
Sorry, I didn't test that.

Reply:
@SmallJoker Won't that cause trouble to people which compile, for example, via SHH or virtual terminal?

Reply:
Right, I see. There's still `FindX11` in CMake - but a reliable detection doesn't seem to be possible with that either.